HPXMLtoOpenStudio/tests/test_hvac.rb

```diff
@@ -754,11 +869,11 @@ def test_heat_pump_advanced_defrost
     assert_equal(1, model.getCoilHeatingDXMultiSpeeds.size)
     htg_coil = model.getCoilHeatingDXMultiSpeeds[0]
     # q_dot smaller than backup capacity
-    _check_advanced_defrost(model, htg_coil, 4747.75, 4747.75, backup_fuel, 0.06667, 1215.05)
+    _check_advanced_defrost(model, htg_coil, 4747.75, 4747.75, backup_fuel, 0.06667, 1067.05)
```

Comment:
The change here is caused by watts_per_cfm used in the p_dot calculation changed due to the test file location change.